recipes/dylib/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Please, add an empty new line at the end of the file, same for other files. We had a past incident with git where resulted in conflicts in the CI because mixing file with vs without empty line.